modules/core/shared/src/main/scala/io/circe/Json.scala

```diff
@@ -168,6 +168,15 @@ sealed abstract class Json extends Product with Serializable {
       case _ => that
     }
 
+  /**
+   *Equality method that does not consider field order or null
```

Comment:
I think the "dropNullValues" has nothing to do with the order of the fields.
In any case (with or without dropNullValues) the equality of JSON does not care about the order.

Reply:
Agreed.  I think the comment is fine either way since its still technically true.  Might be worth adding additional comments to the standard `equals` operator to make it clear.